docs/pre_executed/testing/baselinecls_testing_runtime_config.toml

```diff
@@ -3,50 +3,58 @@ name = "applecider.models.HyraxBaselineCLS.HyraxBaselineCLS"
 
 [model.HyraxBaselineCLS]
 use_probabilities = false
+use_pretrained_weights = true
+pretrained_weights_path = "./pretrained_weights.pth"
```

Comment:
Another approach that we've taken in the past is to consolidate this into a single key. i.e. `pretrained_weights_path = false | <path_to_weights_file>`.  

Then in the baselineCLS class you can check:
```
if config[pretrain_weights_path]:
    # load the weights with the path string.
```

Your approach means that you can leave the path in the config (so you don't have to remember the path) but requires changing (possibly) two config values.

The other approach requires changing only one config value, but requires that you remember the weights file path. 

Either way is just fine - might just want to add a note when it makes it way into the default config.toml file :)

Reply:
Nice, changed it and added a note!